docs/5g-on-helium/cbrs-radios.mdx

```diff
@@ -97,27 +74,33 @@ change frequently.
 
```

Comment:
three > four

Reply:
this > these

Reply:
this whole section was pulled for now, will publish in an additional PR when explorer is updated